app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"""INSERT INTO {self.table_name}
+            (first_name, Last_name) VALUES (?, ?)""",
```

Comment:
not Last_name but last_name, you will get an error when will try to create new entry in db